p2panda-rs/src/entry/entry_signed.rs

```diff
@@ -191,4 +363,41 @@ mod tests {
         let key_value_retrieved = hash_map.get(&entry_first_encoded).unwrap().to_owned();
         assert_eq!(key_value, key_value_retrieved)
     }
+
+    proptest! {
+        #[test]
+        fn non_standard_strings_dont_crash(ref s in "\\PC*") {
+            println!("{s}");
```

Comment:
Is this `println` statement required for this test?

Reply:
I just wanted to see the funny strings.....